config/routes 2.rb

```diff
@@ -0,0 +1,3 @@
+Rails.application.routes.draw do
+  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+end
```

Comment:
Nitpicking... I see this file named `routes 2.rb`. This seems like a mistake? If it is a mistake, you all should feel free to delete that file before submission ;)